packages/scaffolder-cli/src/commands/index.ts

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright 2020 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { Command } from 'commander';
+import { assertError } from '@backstage/errors';
+
+export function registerCommands(program: Command) {
+  program
+    .command('generate <templateDirectory>')
```

Comment:
I think it's worth adding option here to be able to configure the following:

- Backstage base URL, i.e. `http://localhost:7007` in this example case, and maybe even default to that.
- Template data, probably a `--values` or `--data` option where you can either pass inline JSON, detected by a leading `{`, or a file path to a YAML file. Since JSON is a subset of YAML that'll be supported too.
- `template.yaml` location as a relative file path, defaulting to `./template.yaml`

In addition we also need to be able to pass an auth token, but avoid placing that as an option as the tokens can be long-lived and might get stored in shell history. Instead I'd probably opt for a `TOKEN` env var that is read and used if present.